### ISSUE TITLE
Cannot Get Host Details with SDK

### ISSUE THREAD

**Issue by jandersonbt** [tenable/Tenable.io-SDK-for-Python#57]
I am currently using the SDK to try and generate custom reports.  I could be missing something, but it seems that getting the details for a specific host are missing from the API.  Any chance this could be added in?  Does it already exist and I'm missing it?  If the API call isn't part of the API currently, was there a specific reason it was left out?   Thanks!

https://cloud.tenable.com/api#/resources/scans/host-details

**Comment by pierresc** [comment]
+Alan Fung

**Comment by tenableAndrew** [comment]
@jandersonbt There is an update being worked on now that includes the call you mention. We expect it to be released in the next day or so. 

In the meantime you could try something like this to get around it:
```
from tenable_io.client import TenableIOClient
from tenable_io.api.models import BaseModel

host_details = {}

client = TenableIOClient()
scans = [scan.id for scan in client.scans_api.list().scans]

# For each scan, find all hosts and add them to a dictionary keyed by scan_id
for scan in scans:
    scan_hosts = []
    hosts = [host['host_id'] for host in client.scans_api.details(scan).hosts]
    for host in hosts:
        response = client.get('scans/%(scan_id)s/hosts/%(host_id)s',
                                    path_params={'scan_id': scan, 'host_id': host})
        scan_hosts.append(BaseModel.from_json(response.text).info)
    host_details[scan] = scan_hosts

print(host_details)
```  
Which should print something like:
```
{
  "17": [
    {
      "host-ip": "192.168.0.38",
      "host_end": "Thu Sept 10 09:09:28 2017",
      "host_start": "Thu Sept 10 09:06:53 2017",
      "operating-system": "Microsoft Windows Server 2012 R2 Standard",
      "host-fqdn": "testserverms1"
    }
  ],
  "19": [
    {
      "host-ip": "192.168.0.42",
      "host_end": "Thu Sept 11 11:29:24 2017",
      "host_start": "Thu Sept 11 11:26:59 2017",
      "operating-system": "Microsoft Windows 8.1",
      "host-fqdn": "msworkstation1"
    }
  ]
}
```

**Comment by jandersonbt** [comment]
Thanks so much for the update!  And thank you for the quick fix!

**Comment by alanfung** [comment]
@jandersonbt thank you for reporting this issue, we have addressed this in the 0.3.0 release.

`TenableIOClient().scans_api.host_details(scan_id, host_id)`
